tiflash/use-tiflash.md

```diff
@@ -79,6 +79,65 @@ In the result of above statement:
 * `AVAILABLE` indicates whether the TiFlash replicas of this table is available or not. `1` means available and `0` means unavailable. Once the replicas become available, this status does not change. If you use DDL statements to modify the number of replicas, the replication status will be recalculated.
 * `PROGRESS` means the progress of the replication. The value is between `0.0` and `1.0`. `1` means at least one replica is replicated.
 
+## Create TiFlash replicas for databases
+
+Similar to creating TiFlash replicas for tables, you can send a SQL statement to TiDB through a MySQL client to create a TiFlash replica for all tables in a specific database:
+
+{{< copyable "sql" >}}
+
+```sql
+ALTER DATABASE db_name SET TIFLASH REPLICA count
+```
+
+In this statement, `count` indicates the number of replicas. When the value is `0`, replicas are deleted.
+
+Examples:
+
+- Create two replicas for all tables in the database `tpch50`:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    ALTER DATABASE `tpch50` SET TIFLASH REPLICA 2
+    ```
+
+- Delete TiFlash replicas created for the database `tpch50`:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    ALTER DATABASE `tpch50` SET TIFLASH REPLICA 0
+    ```
+
+> **Note:**
+>
+> - This statement actually performs a series of DDL operations, which is resource-intensive. If the statement is interrupted during the execution, executed operations are not rolled back and unexecuted operations do not continue.
+>
+> - After executing the statement, do not set the number of TiFlash replicas or DDL operations on this database until **all tables in this database are replicated**. Otherwise, unexpected results may occur, which includes:
```

Comment:
```suggestion
> - After executing the statement, do not set the number of TiFlash replicas or perform DDL operations on this database until **all tables in this database are replicated**. Otherwise, unexpected results might occur, which include:
```